pages/project/read/index.js

```diff
@@ -30,26 +30,20 @@ module.exports = settings => {
       licenceHolderId: req.project.licenceHolderId,
       establishment: req.establishment.id
     };
-    Promise.resolve()
-      .then(() => req.user.can('project.update', params))
-      .then(canUpdate => {
+    Promise.all([
+      req.user.can('project.update', params),
+      req.user.can('project.revoke', params)
+    ])
```

Comment:
This is because ASRU can't update but they _can_ revoke, right?

Reply:
yeah, we were only calling permissions service for revoke if the task was `editable` before, which didn't make sense for asru users